app/src/docker/docker-compose.yml

```diff
@@ -190,3 +191,29 @@ services:
     networks:
       - internal
 
+  vro-console:
+    image: va/abd_vro-console:latest
+    profiles: ["debug"]
```

Comment:
The console container only starts if the docker-compose `debug` profile is enabled: `export COMPOSE_PROFILES=debug`